src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -37,9 +38,15 @@
  */
 public class XdocsTemplateSink extends XdocSink {
```

Comment:
I assume we need https://github.com/checkstyle/checkstyle/pull/13696 to prove this will be fixed in the CI.

I would expect that PR to be merged first and then this PR should show the removal of the temporary overrides.

Reply:
@kalpadiptyaroy, please cherry-pick this commit to https://github.com/checkstyle/checkstyle/pull/13696 to show clearly it works.
as maintiners agree, and clearly see it works even for next PR, we will merge this Pr.

Reply:
ok. on it!